haproxy/metadata.csv

```diff
@@ -4,10 +4,10 @@ haproxy.backend.agg.server.check.status,gauge,,,,[OpenMetrics V1 and V2] Backend
 haproxy.backend.agg.check.status,gauge,,,,[OpenMetrics V1 and V2] Backend's aggregated gauge of servers' state check status (since >= 2.4).,0,haproxy,,
 haproxy.backend.agg.server.status,gauge,,,,[OpenMetrics V1 and V2] Backend's aggregated gauge of servers' status (since >= 2.4).,0,haproxy,,
 haproxy.backend.backup.servers,gauge,,,,[OpenMetrics V1 and V2] Current number of backup servers.,0,haproxy,,
-haproxy.backend.bytes.in.total,count,,byte,,[OpenMetrics V1] Current total of incoming bytes. By default submitted as count if using prometheus,0,haproxy,,
-haproxy.backend.bytes.in.count,count,,byte,,[OpenMetrics V2] Current total of incoming bytes. By default submitted as count if using prometheus,0,haproxy,,
-haproxy.backend.bytes.out.total,count,,byte,,[OpenMetrics V1] Current total of outgoing bytes. By default submitted as count if using prometheus,0,haproxy,,
-haproxy.backend.bytes.out.count,count,,byte,,[OpenMetrics V2] Current total of outgoing bytes. By default submitted as count if using prometheus,0,haproxy,,
+haproxy.backend.bytes.in.total,count,,byte,,[OpenMetrics V1] Current total of incoming bytes. By default, submitted as count if using Prometheus.,0,haproxy,,
```

Comment:
```suggestion
haproxy.backend.bytes.in.total,count,,byte,,"[OpenMetrics V1] Current total of incoming bytes. By default, submitted as count if using Prometheus.",0,haproxy,,
```

You need to add extra `"` if you add commas in the description because it's a csv file

Reply:
You'll have to update the other lines too